File: python/60a_1092B_TeamsForming.py
def main():
    n = int(input())
    arr = list(map(int, input().split()))

    arr.sort()
    cnt = 0
    for i in range(0,n,2):
        cnt += (arr[i+1] - arr[i])
    print(cnt)

if __name__ == "__main__":
    main()

File: cpp/60a_1092B_TeamsForming.cpp
#include<bits/stdc++.h>

using namespace std;

int main(){
    ios::sync_with_stdio(0);
    cin.tie(0);

    int n, cnt, temp;
    vector<int> arr;

    cin>>n;
    for(int i = 0; i < n; i++){
        cin>>temp;
        arr.push_back(temp);
    }
    sort(arr.begin(), arr.end());

    cnt = 0;
    for(int i = 0; i < n; i += 2){
        cnt += (arr[i+1] - arr[i]);
    }

    cout<<cnt;
    return 0;
}